cypress/e2e/shared/explorer.test.ts

```diff
@@ -530,9 +528,9 @@ describe('DataExplorer', () => {
     it('shows the empty state when the query returns no results', () => {
       cy.getByTestID('time-machine--bottom').within(() => {
         cy.getByTestID('flux-editor').should('be.visible')
-          .monacoType(`from(bucket: "defbuck"{rightarrow}
-  |> range(start: -10s{rightarrow}
-  |> filter(fn: (r{rightarrow} => r._measurement == "no exist"{rightarrow}`)
+          .monacoType(`from(bucket: "defbuck")
+  |> range(start: -10s)
+  |> filter(fn: (r) => r._measurement == "no exist")`)
```

Comment:
when I tried this, I think `monacoType` ended up typing two closing parentheses. I think the `rightarrow` is here to deal with autocomplete. Just something to be aware of - I know you're changing `monacoType`.

Reply:
Strange. It gave me only 1 closing parenthesis, basically typing exactly what's in the string. Here is a video of it running on Chrome.


https://user-images.githubusercontent.com/10736577/156660428-34d2ec7b-2ad1-4289-b8fd-966d5ce3c308.mp4

Reply:
Same in Firefox, types exactly what I have in the string.


https://user-images.githubusercontent.com/10736577/156660966-e9a97d55-b463-46c1-b683-a661216decf2.mp4